CRM/Core/BAO/Navigation.php

```diff
@@ -337,6 +345,33 @@ public static function buildNavigation() {
     return $navigationString;
   }
 
+  /**
+   * Items are more or less ordered by weight (since buildNavigationTree
+   * retreived them in order. However, any items added by the hook
```

Comment:
Want to finish this thought?

Reply:
thanks @colemanw - thought now finished.